content/source/docs/enterprise/api/run.html.md

```diff
@@ -29,6 +29,7 @@ A run performs a plan and apply, using a configuration version and the workspace
 
 - `id` (`string: <required>`) - specifies the workspace ID to run
 - `is-destroy` (`bool: false`) - specifies if this plan is a destroy plan, which will destroy all provisioned resources.
+- `message` (`string: <optional`) - specifies the message to be associated with this run.
```

Comment:
Hey, while you're in this file... could I get you to reformat this method to follow `_template.md`? 😄